core/shared/mem-alloc/mem_alloc.cmake

```diff
@@ -24,6 +24,10 @@ if (WAMR_BUILD_GC_CORRUPTION_CHECK EQUAL 0)
     add_definitions (-DBH_ENABLE_GC_CORRUPTION_CHECK=0)
 endif ()
 
+if (DEFINED WAMR_BUILD_GC_HEAP_SIZE_DEFAULT)
+    add_definitions ("-DGC_HEAP_SIZE_DEFAULT=${WAMR_BUILD_GC_HEAP_SIZE_DEFAULT}")
+endif ()
```

Comment:
Please include the description and instructions for using this new compilation option in the document, doc/build_wamr.md.